src/app/teams/teams.component.html

```diff
@@ -41,7 +41,12 @@ <h3>
         <mat-cell *matCellDef="let element">
           <ng-container [ngSwitch]="element.userStatus" *ngIf="user.isUserAdmin || user.roles.length">
             <ng-container *ngSwitchCase="'member'">
-              <button mat-raised-button color="primary" class="margin-lr-3" (click)="toggleMembership(element.doc, true, element.membershipDoc)"><mat-icon>remove_circle</mat-icon> Leave</button>
+              <span *ngIf="user._id === element.doc.createdBy; else leaveButton">
+                <button mat-raised-button color="primary" class="margin-lr-3" (click)="toggleDeleteMembership(element.doc)"><mat-icon>delete_forever</mat-icon> Delete</button>
```

Comment:
`toggleDeleteMembership` name should be changed as it deletes team not toggledeletion

Reply:
i have changed the name now.. thanx in advance for your review